### ISSUE TITLE
Screen Flickering when SEB is launch

### ISSUE THREAD

**Issue by vanie76** [SafeExamBrowser/seb-win-refactoring#520]
Student try to launch SEB, however the screen was flickering when SEB is launching
Only happens for those using Huawei laptop
**Screenshots**

**Version Information**
 - OS: Windows 10
 - SEB-Version [3.4.1.505

**Additional Context**

https://user-images.githubusercontent.com/87995554/203266475-93786784-29a4-4a94-af4b-d77b4e01bc4d.mp4

**Comment by vkatavic** [comment]
We've been having the same issue with some of our students. I haven't checked who the manufacturer of their laptops is. I'll provide a video too

**Comment by dbuechel** [comment]
There clearly is some sort of software or utility interfering with the kiosk mode of SEB, namely attempting to reactivate the default Windows desktop. You should see log message stating the exact details in the client application log.

There was an issue where this happened with Cisco Webex (see #333), so I highly suspect a similar meeting or desktop sharing / recording software.

**Comment by michaelhq** [comment]
On two of my computers with current Windows 11, the problem also occurred. After closing all potential programs that could have caused the issue, the issue still occurred. Versions 3.4.1, 3.4.0 and 3.3.2 of SEB showed the same flickering. My final guess was the antivirus protection program I had installed. ESET Internet Security is installed on both computers. However, temporarily terminating the protection did not help because it could not be terminated completely (which is also better this way). So I uninstalled the antivirus program completely and the flickering was gone. 
After I reinstalled the antivirus program, the flickering no longer occurred. I suspect that the reinstallation set the configuration of the program to the default setting and certain parts of the program are now running on the latest version. Unfortunately, I could not find any reference to possible issues with kiosk mode or multiple desktops in the antivirus knowledge base. 
On computers that are “only” protected with Windows Defender, no flickering had occurred.
